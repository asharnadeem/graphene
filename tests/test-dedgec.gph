int main(){
  node<string> a;
  node<string> b;
  a ->["5"] b;
  print(a.edges[0].weight);
  print(a.edges[0].dest);
  return 0;
}
