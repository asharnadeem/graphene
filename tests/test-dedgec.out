5
id: 0, val: (null), edges: 1
